docs/migrating-from-0.5.md

```diff
@@ -2,7 +2,122 @@
 
 Genkit 0.9 introduces a number of breaking changes alongside feature enhancements that improve overall functionality. If you have been developing applications with Genkit 0.5, you will need to update your application code when you upgrade to the latest version. This guide outlines the most significant changes and offers steps to migrate your existing applications smoothly.
 
-## 1. CLI Changes
+## Quickstart guide
+
+The following steps will get you up and running on Genkit 0.9 quickly. Read more information about these changes in the detailed [Changelog](#changelog) below.
+
+### 1. Install the new CLI
+
+* Uninstall the old CLI
+
+```posix-terminal
+npm uninstall -g genkit
```

Comment:
I think we want to `npm uninstall -g genkit && npm uninstall genkit` to remove it both locally and from the `devDependendencies` section of the package

Reply:
Done.